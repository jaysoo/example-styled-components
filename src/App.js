import React from 'react';
import styled from 'styled-components';
import Message from './Message';
import Container from './Container';

const App = () => (
  <Container>
    <Message className="foo">
      Hello
    </Message>
    <Message className="bar" color="rebeccapurple">
      World!
    </Message>
  </Container>
);

export default App;
