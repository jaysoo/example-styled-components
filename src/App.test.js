import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import App from './App';

it('renders', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
});
