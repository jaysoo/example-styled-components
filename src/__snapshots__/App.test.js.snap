// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c1 {
  font-size: 24px;
  color: palevioletred;
}

.c5 {
  font-size: 24px;
  color: rebeccapurple;
}

.c6 {
  font-weight: normal;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: Helvetica, sans-serif;
}

.c6 .{
  margin: 3px;
}

<div
  className="c6"
>
  <p
    className="c0 sc-bdVaJa c1"
  >
    Hello
  </p>
  <p
    className="c4 sc-bdVaJa c5"
  >
    World!
  </p>
</div>
`;
